web/components/admin/ApplicantList/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ApplicantList extends React.Component {
+  render() {
+    const { applicants, selectedApplicantId, onApplicantClick, className } = this.props;
+    let applicantNumber = 0;
+
+    if (!applicants) return null;
+    return (
+      <div className={`applicant-list ${className}`}>
+        {applicants.map((applicant) => {
+          const { id, firstName, lastName, email, finalScore } = applicant;
+          applicantNumber += 1;
+          return (
+            <div
+              className={`applicant flex ${selectedApplicantId === id ? 'selected' : ''}`}
+              key={id}
+              data-applicant-id={id}
+              onClick={onApplicantClick}
```

Comment:
What do you think about passing in ID in the `onClick` function instead of relying on `data-applicant-id`?

i.e. `onClick={() => onApplicantClick(id)}`

I think especially since we don't have unit testing anymore, it's not really obvious that `data-applicant-id` is a necessary attribute for `onApplicantClick` to work.